worlds/mlss/Rules.py

```diff
@@ -546,26 +646,55 @@ def set_rules(world: "MLSSWorld", excluded):
         add_rule(
             world.get_location(LocationName.GwarharLagoonFirstUnderwaterAreaRoom2CoinBlock),
             lambda state: StateLogic.canDash(state, world.player)
-            and (StateLogic.membership(state, world.player) or StateLogic.surfable(state, world.player)),
+                          and (StateLogic.membership(state, world.player) or StateLogic.surfable(state, world.player)),
         )
         add_rule(
             world.get_location(LocationName.JokesEndSecondFloorWestRoomCoinBlock),
             lambda state: StateLogic.ultra(state, world.player)
-            and StateLogic.fire(state, world.player)
-            and (
-                StateLogic.membership(state, world.player)
-                or (StateLogic.canDig(state, world.player) and StateLogic.canMini(state, world.player))
-            ),
+                          and StateLogic.fire(state, world.player)
+                          and (
+                                  StateLogic.membership(state, world.player)
+                                  or (StateLogic.canDig(state, world.player) and StateLogic.canMini(state,
+                                                                                                    world.player))
```

Comment:
Might be better to split this line so that `StateLogic.canMini(state, world.player))` is on one line